package.json

```diff
@@ -78,7 +81,7 @@
     "download": "node ./bin/download-yarn",
     "prepare": "npm run clean && npm run build && npm run download",
     "release": "np",
-    "test": "npm run download && jest -i && flow && standard",
+    "test": "npm run download && jest && flow && standard",
```

Comment:
they still should be run in series though right? If the tests are doing things like installing and uninstalling plugins we can't be running things in parallel